tests/cli-v2.rs

```diff
@@ -1374,3 +1374,53 @@ fn check_pgp_keys() {
         );
     })
 }
+
+#[test]
+fn install_allow_downgrade() {
+    clitools::setup(Scenario::MissingComponent, &|config| {
+        let trip = TargetTriple::from_build();
+
+        // this dist has no rls and there is no newer one
+        set_current_dist_date(config, "2019-09-14");
+        expect_ok(
+            config,
+            &[
+                "rustup",
+                "toolchain",
+                "install",
+                "nightly",
+                "--no-self-update",
+            ],
+        );
```

Comment:
After this, I'd add an expect to check the rustc version is the version we expect (i.e. one without rls)

Reply:
Yes, I add that. It is though like a sanity check to find out if our test setup is based on the right assumptions.